runtime/protocol/http-client-engines/http-client-engine-okhttp/jvm/test/aws/smithy/kotlin/runtime/http/engine/okhttp/OkHttpResponseTest.kt

```diff
@@ -71,114 +67,4 @@ class OkHttpResponseTest {
 
         assertContentEquals(content, actualBody)
     }
-
-    @Test
-    fun testCallContextCancelsWriter() = runTest {
-        // call context cancelled should result in no children left
```

Comment:
**Question**: This test is no longer necessary because everything's synchronous now?

Reply:
Yeah we aren't launching any background coroutines so there is no cancellation to worry about here. Callers will be reading the body directly rather than proxied via channel